_layouts/default.html

```diff
@@ -133,6 +133,7 @@
 	<a href="https://github.com/endless-sky/endless-sky">GitHub</a><br />
 	<a href="https://discord.gg/ZeuASSx">Discord</a><br />
 	<a href="https://store.steampowered.com/app/404410/Endless_Sky/">Steam</a><br />
+	<a href="https://www.gog.com/en/game/endless_sky">GOG</a><br />
```

Comment:
Maybe use 
```suggestion
	<a href="https://www.gog.com/game/endless_sky">GOG</a><br />
```
for a user-localized page